test/hummingbot/connector/exchange/bitmart/test_bitmart_api_order_book_data_source.py

```diff
@@ -344,21 +351,28 @@ def _trade_ws_messsage(self):
         }
         return ujson.dumps(resp)
 
-    @patch('websockets.connect', new_callable=AsyncMock)
+    @patch('aiohttp.ClientSession.ws_connect', new_callable=AsyncMock)
     def test_listen_for_trades(self, mock_ws):
-        msg_queue: asyncio.Queue = asyncio.Queue()
+        msg_queue: asyncio.Queue = BitmartAPIOrderBookDataSourceUnitTests.QueueMock()
+
         mock_ws.return_value = self.mocking_assistant.create_websocket_mock()
         mock_ws.close.return_value = None
 
         # Add message to be processed after subscriptions, to unlock the test
-        self.mocking_assistant.add_websocket_text_message(mock_ws.return_value, self._trade_ws_messsage())
+        self.mocking_assistant.add_websocket_aiohttp_message(mock_ws.return_value, bitmart_utils.compress_ws_message(self._trade_ws_messsage()), message_type=aiohttp.WSMsgType.BINARY)
```

Comment:
The line is over 120 characters (for these ones, you can use the IDE to your advantage — here's how for [VSCode](https://dev.to/adamlombard/vscode-setting-line-lengths-in-the-black-python-code-formatter-1g62) and here's for [PyCharm](https://stackoverflow.com/questions/17319422/how-do-i-set-the-maximum-line-length-in-pycharm))